data_structures/binary_search_tree.ts

```diff
@@ -205,7 +206,7 @@ export class BinarySearchTree<T> implements Iterable<T> {
    */
   static from<T, U, V = undefined>(
     collection: ArrayLike<T> | Iterable<T> | BinarySearchTree<T>,
-    options: {
```

Comment:
This is wrong. When this overload is used, `options` should be specified. The overload that matches when `options` is not  specified is the first one.

Reply:
Apologies. Mistake.